core/src/main/java/co/aikar/commands/ACFPatterns.java

```diff
@@ -49,17 +49,28 @@
     public static final Pattern EQUALS = Pattern.compile("=");
     public static final Pattern FORMATTER = Pattern.compile("<c(?<color>\\d+)>(?<msg>.*?)</c\\1>", Pattern.CASE_INSENSITIVE);
     public static final Pattern I18N_STRING = Pattern.compile("\\{@@(?<key>.+?)}", Pattern.CASE_INSENSITIVE);
-
-
-
-    private ACFPatterns() {}
     @SuppressWarnings("Convert2MethodRef")
     static final Map<String, Pattern> patternCache = ExpiringMap.builder()
-            .maxSize(200)
-            .expiration(1, TimeUnit.HOURS)
-            .expirationPolicy(ExpirationPolicy.ACCESSED)
-            .build();
+                                                                .maxSize(200)
```

Comment:
Drop the .builder to next line so we can keep the rest aligned the same, I really hate super far indentations.